### DISCUSSION TITLE
sail in ubuntu installed on windows (wsl2) gives me -bash: ./vendor/bin/sail: Permission denied

### DISCUSSION THREAD

**Discussion by giacomomasseron** [laravel/framework#49983] [Q&A]
Hi,  
I installed ubuntu on windows, I installed a laravel project with sail, but when I execute the command (inside the machine):

    ./vendor/bin/sail up

I get this error:  

    -bash: ./vendor/bin/sail: Permission denied

There are no more logs to read, I have no idea how try to solve it.

**Comment by henzeb** [comment]
`chmod +x ./vendor/bin/sail` would give it execution permission.
